aries_cloudcontroller/controllers/connections.py

```diff
@@ -75,18 +68,34 @@ async def create_invitation(
         self,
         alias: str = None,
         auto_accept: bool = None,
-        public: str = None,
-        multi_use: str = None,
+        public: bool = None,
+        multi_use: bool = None,
+        invite_options: dict = None,
     ):
         params = {}
         if alias:
             params["alias"] = alias
-        if auto_accept:
+        if auto_accept in [True, False]:
             params["auto_accept"] = auto_accept
-        if public:
+        if public in [True, False]:
             params["public"] = public
-        if multi_use:
+        if multi_use in [True, False]:
             params["multi_use"] = multi_use
+        if invite_option:
+            """A dictionary of the form:
+            {
+                "mediation_id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+                "metadata": {},
+                "recipient_keys": [
+                    "H3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV"
+                ],
+                "routing_keys": [
+                    "H3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV"
+                ],
+                "service_endpoint": "http://192.168.56.102:8020"
+            }
+            """
+            params['body'] = invite_options
```

Comment:
Is this right? I don't think so.

I think this body should be sent as json_data

Reply:
Yeah good shout! Silly me. It's changed now.